src/components/TaskList.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <fieldset class="cfl-task-list">
+    <legend>Volunteer Opportunities</legend>
+    <p v-if="!tasks.length" class="cfl-no-tasks-info">
+      We don't have any specific opportunities to show you right now, but if
+      you submit your contact information we'll find a way you can help!
```

Comment:
We'll want to make these messages easily configurable at some point

Reply:
I had the same thought and the [@intlify/vue-i18n](https://github.com/intlify/vue-i18n-next) plugin may work well for this.